cli/azd/extensions/azd.internal.registry/internal/cmd/root.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewRootCommand() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:           "azd registry <command> [options]",
```

Comment:
Just wondering if the `azd` prefix is expected here. Feel free to address this separately.

I could imagine that the extension framework automatically appends "azd" to the `Use` here. For examples, it makes sense that the extension needs to include the full invocation path.

Reply:
Most of this comes down to the correct usage of cobra or other CLI toolkits for constructing the correct help experience for the custom extension commands.  The goal is for the help experience to still show `azd <extension-namespace> <command> [options]` usage patterns for commands but I just haven't figured out the correct configuration here.

This is a great question that we will figure out over time and maybe the extension framework can help compose this together better.